modules/learn/pages/data/distributed-transactions.adoc

```diff
@@ -0,0 +1,315 @@
+
+== Deployment Considerations
+
+If using a single node cluster (for example, during development), then note that the default number of replicas for a newly created bucket is 1. If left at this default, then all durable Key-Value writes, which are used by transactions, will fail with a DurabilityImpossibleException. This setting can be changed via xref:manage:manage-buckets/create-bucket.adoc#couchbase-bucket-settings[GUI] or xref:cli:cbcli/couchbase-cli-bucket-create.adoc#options[command line]. If changed on a bucket that already exists, the server needs to be rebalanced.
+
+Use of transactions requires Network Time Protocol (NTP) to be used to synchronize time across all cluster-nodes. See xref:install:synchronize-clocks-using-ntp.adoc[Clock Sync with NTP] for details.
+
+== Settings and Parameters
+
+Transactions can be configured using a number of settings and request-level parameters. 
+
+[cols="2", options="header"]
+|===
+|Parameter
+|Description
+
+|Durability level
+|xref:java-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc#configuration
+
+|Scan consistency
+|xref:settings:query-settings.adoc#transactional-scan-consistency
+
+|Request-level Query parameters
+|Request-level parameters when using queries within transactions. See xref:n1ql:n1ql-language-reference/transactions.adoc#settings-and-parameters[] for details.
+
+|Transaction expiry timer
+|Configures how long a transaction should last before it is rolled back.
+The transaction expiry timer (which is configurable) will begin ticking once the transaction starts, and is not paused while the transaction is in a deferred state.
+The default value is 15 seconds. Within this timeframe, if there are concurrency or node issues, a combination of wait and retry operations are used until the transaction reaches this time.
+For more information, see xref:java-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc#error-handling[].
+
+|tximplicit
+|Specifies that a DML statement is a singleton transaction. By default, it is set to false.
+See xref:settings:query-settings.adoc#tximplicit[tximplicit] for details.
+
+|kvtimeout
+|Specifies the maximum time to wait for a KV operation before timing out. The default value is 2.5s. See xref:settings:query-settings.adoc#kvtimeout[kvtimeout] for details.
+
+|atrcollection
+|Specifies the collection where the active transaction record (ATR) is stored. The collection must be present. If not specified, the ATR is stored in the default collection in the default scope in the bucket containing the first mutated document within the transaction. See 
+xref:settings:query-settings.adoc#atrcollection_req[atrcollection] for details.
+|===
+
+== Related Topics
+
+* xref:java-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc[]
+* xref:cxx-txns:distributed-acid-transactions-from-the-sdk.adoc
+* xref:dotnet-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc
```

Comment:
Do these xrefs need a []
added?
I don't tend to use them without link text, so I'm unsure.

Reply:
If we don't specify the label, it uses the title off the linked page.

Reply:
Yes, gathered that - was more the lack then of an _empty_ [], which was used for the first of these bullets.
Is line 313 correct, or the others without []  ?